cli.ts

```diff
@@ -44,6 +50,15 @@ sade('lighthouse-parade <url> [dataDirectory]', true)
     'Control the maximum number of ligthhouse reports to run concurrently',
     os.cpus().length - 1
   )
+  .option('--max-depth', 'Control the maximum depth of crawled links')
+  .option(
+    '--path-must-match',
+    'Specify a glob (in quotes) for paths to match. Links to non-matched paths will not be crawled. This flag can be specified multiple times to allow multiple paths'
```

Comment:
I wonder if we should add language to clarify that the initial entry point URL is an exception. Correct? If I do `lighthouse-parade https://cloudfour.com --path-must-match "/foofoodoesntexist"` I will _still_ receive one report for the homepage, right? I can't tell if that would be obvious or helpful.